src/deltarun/delta-run-service.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+
+export function getDeltaRunTarget(sfgecachepath:string, savedFilesCache:Set<string>): string[] {
```

Comment:
Conceptually, it feels weird for this to be a utility method that accepts the `Set`, instead of having a `class` that encloses the `Set` and exposes a `getDeltaRunTarget` method.
But broadly speaking, I think this is fine.